user_guide_src/source/changelogs/v4.3.0.rst

```diff
@@ -27,6 +27,7 @@ Others
 ------
 
 - The ``spark`` file has been changed due to a change in the processing of Spark commands.
+- ``InvalidArgumentException`` that is a kind of ``LogicException`` in ``BaseBuilder::_whereIn()`` is not suppressed by the configuration. Previously if ``CI_DEBUG`` was true, the exception was suppressed.
```

Comment:
I think you mean `CI_DEBUG` is false, the exception was suppressed.

Reply:
Yes! Thanks.